### PR TITLE
Polish holidays

### PR DESCRIPTION
**Pull request by kamilogorek**
~There are also 4 movable holdays, but there's no way to mark them right now.~

Fixed.

### CODE REVIEW

File: holidays/pl/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "08-15",
+    "11-01",
+    "11-11",
+    "12-25",
+    "12-26"
+  ],
+  region: []
+};
+
+function pentecost() {
```
Comment (WebReflection) [inline holidays/pl/index.js]: the signature wants Y

File: holidays/pl/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  var date = new Date(common.easter(Y).getTime());
+  date.setDate(date.getDate() + 49);
+  return date;
+}
+
+function corpusChristi() {
```
Comment (WebReflection) [inline holidays/pl/index.js]: the signature wants Y

**Comment by kamilogorek** [comment]
Regarding religious holidays, yes, it'll most likely end up in `common.js` sooner or later. Didn't want to move it there, as no other location is using them just yet.

**Comment by WebReflection** [comment]
@kamilogorek pentecost and pentecostMonday are now in utils.js together with other utilities ;-)

**Pull request merged**